frontend/backend/user/user.ts

```diff
@@ -103,7 +103,18 @@ export const getUser = async (wallet: string) => {
     }
   });
 
-  if (!res) return { error: ERRORS.USER_NOT_FOUND };
+  if (!res) {
```

Comment:
@francisco-leal @0xForkHunter for some reason i can't perform an OR query here (lint errors), can you help?